src/frontend/Sider/Personoversikt/Behandlingsoversikt/OpprettNyBehandling/OpprettNyBehandlingModal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+
+import { Button, Select, VStack } from '@navikt/ds-react';
+
+import OpprettKlageBehandling from './OpprettKlageBehandling';
+import { useApp } from '../../../../context/AppContext';
+import { ModalWrapper } from '../../../../komponenter/Modal/ModalWrapper';
+import {
+    BehandlingType,
+    behandlingTypeTilTekst,
+} from '../../../../typer/behandling/behandlingType';
+import { RessursStatus } from '../../../../typer/ressurs';
+
+interface OpprettKlageRequest {
+    mottattDato: string;
+    klageGjelderTilbakekreving: boolean;
+}
+
+const OpprettNyBehandlingModal = () => {
+    const { request } = useApp();
+
+    const [visModal, settVisModal] = useState(false);
+    const [behandlingtype, settBehandlingtype] = useState<BehandlingType>();
+    const [klageGjelderTilbakekreving, settKlageGjelderTilbakekreving] = useState<boolean>(false);
+    const [kravMottattDato, settKravMottattDato] = useState('');
+    const [feilmelding, settFeilmelding] = useState<string>();
+
+    const opprettKlage = (data: OpprettKlageRequest) => {
+        request<null, OpprettKlageRequest>(`/tilleggsstonader-klage`, 'POST', data).then(
+            (response) => {
+                if (response.status === RessursStatus.SUKSESS) {
+                    lukkModal();
+                } else {
+                    settFeilmelding(response.frontendFeilmelding || response.melding);
+                }
+            }
+        );
+    };
+
+    const lukkModal = () => {
+        settVisModal(false);
+        settFeilmelding('');
+        settKlageGjelderTilbakekreving(false);
+        settKravMottattDato('');
+        settBehandlingtype(undefined);
+    };
+
+    return (
+        <div className="py-16">
+            <Button variant={'secondary'} onClick={() => settVisModal(true)}>
+                Opprett ny behandling
+            </Button>
```

Comment:
Burde denne knappen skjules i prod frem til det faktisk fungerer å lage en klage?

Reply:
Knappen for innsending er allerede disabled.. Så spørsmålet er hvor mye man trenger å skjule